OWNERS_ALIASES

```diff
@@ -0,0 +1,22 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
+  sig-cluster-lifecycle-leads:
+    - lukemarsden
+    - luxas
+    - roberthbailey
+    - timothysc
+  cluster-api-admins:
+    - justinsb
+    - krousey
+    - luxas
```

Comment:
I agree that he should be in the sig leads and part of approvers. I think he was a kube-deploy admin, but not very active in the cluster-api aspect of it. @luxas, does that seem accurate?